--- site-packages/torchvision/transforms/functional.py	2020-01-25 20:43:38.870781898 +0800
+++ site-packages/torchvision/transforms/functional.py	2020-01-25 20:45:11.233776827 +0800
@@ -2,7 +2,7 @@
 import torch
 import sys
 import math
-from PIL import Image, ImageOps, ImageEnhance, PILLOW_VERSION
+from PIL import Image, ImageOps, ImageEnhance, __version__
 try:
     import accimage
 except ImportError:
